devlab/generate_load.py

```diff
@@ -12,23 +12,24 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import ConfigParser
 import argparse
 import itertools
-import time
 import json
+import logging
 import os
-import yaml
-import ConfigParser
+import time
+from logging.config import dictConfig
 
+import yaml
 from keystoneclient import exceptions as ks_exceptions
 from neutronclient.common import exceptions as nt_exceptions
 from novaclient import exceptions as nv_exceptions
 
-from base import BasePrerequisites
-from cleanup import CleanEnv
-import config as conf
-import logging
-from logging.config import dictConfig
+import devlab.tests.config as conf
+from devlab.tests.base import BasePrerequisites
```

Comment:
I think that would be better to import module, not class. (smth like "from devlab.tests import base')